### ISSUE TITLE
[Feature] Export Lora Adapters as GGML

### ISSUE THREAD

**Issue by rmarnold** [ml-explore/mlx-examples#816]
lama.cpp dropped support for converting lora to ggml, it would be very useful if we could use adapters with llama.cpp instead of fusing or merging the fine tuned model.

**Comment by awni** [comment]
Can you say more about what you are looking for?

Is it a separate GGUF file which contains the adapters. Then you can load the base model GGUF as well as the adapter GGUF in llama.cpp? Does it llama.cpp support that?

**Comment by rmarnold** [comment]
@awni, Yes, in llama.cpp you can specify --lora ggml-adapter-model.bin. The issue is that after training, mlx_ml outputs adapter.safetensors, which llama.cpp does not recognize. I know for certain that the supported output format is ggml, but I have read there has been work to support gguf.
